app/scripts/modules/core/src/presentation/hooks/usePollingData.hook.spec.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect } from 'react';
+import { mount } from 'enzyme';
+import { IUseLatestPromiseResult } from './useLatestPromise.hook';
+import { usePollingData } from './usePollingData.hook';
+
+describe('usePollingData hook', () => {
```

Comment:
Because this hook really just delegates down to `useData` and `SchedulerFactory` directly with just a teeny bit of `useEffect` logic thrown in, I didn't feel it was valuable to go deep on tests. These verify the basic assumptions that we'll return a proper result and refresh on an interval.